minigalaxy/ui/window.py

```diff
@@ -32,6 +33,9 @@ def __init__(self, name="Minigalaxy"):
         self.search_string = ""
         self.offline = False
 
+        # Make the DownloadManager able to output errors
+        DownloadManager.window = self
```

Comment:
I feel like we should be able to come up with a better solution for this, but I'll need to think about it. I am thinking of making the Window class a singleton, so you can just call ``self.window = Window()`` in the constructor of the DownloadManager without causing circulair dependencies. Alternatively we could find a way to pass the window to the downloadmanager through the constructor, but for that the downloadmanager is not created in the right place at the moment.

Reply:
Sorry about this, I know this solution was my idea. This one wouldn't stop me from merging, it's more of a note to myself.

Reply:
No worries. I would actually propose to delay this PR until that is finished, since what it fixes already rarely occurs and we then can solve it cleanly.

Also, I can then take a look at disk checking elsewhere where it is relevant (e.g. extracting and final download location come to mind).

Making this PR a draft for the time being.